CHANGELOG.md

```diff
@@ -7,6 +7,185 @@ documentation before upgrading to a new release.
+| icingacli-businessprocess        | --colors                   | icingacli\_businessprocess\_colors       | #9103 |
+| icingacli-businessprocess        | --downtime-is-ok           | icingacli\_businessprocess\_downtimeisok | #9103 |
+| icingacli-businessprocess        | --root-cause               | icingacli\_businessprocess\_rootcause    | #9103 |
+| mem                              | -a                         | mem\_available                           | #9385 |
+| mongodb                          | --disable\_retry\_writes   | mongodb\_disableretrywrites              | #9539 |
+| mongodb                          | --ssl-ca-cert-file         | mongodb\_ssl\_ca\_cert\_file             | #9610 |
+| mysql                            | --extra-opts               | mysql\_extra\_opts                       | #9197 |
+| nrpe                             | -3                         | nrpe\_version\_3                         | #9296 |
+| nrpe                             | -D                         | nrpe\_no\_logging                        | #9016 |
+| nrpe                             | -P                         | nrpe\_payload\_size                      | #9032 |
+| pgsql                            | --extra-opts               | pgsql\_extra\_opts                       | #9197 |
+| postgres                         | $PGCONTROLDATA (env. var.) | postgres\_pgcontroldata                  | #8929 |
+| postgres                         | --datadir                  | postgres\_datadir                        | #8924 |
+| postgres                         | --language                 | postgres\_language                       | #8924 |
+| postgres                         | --perflimit                | postgres\_perflimit                      | #8924 |
+| ssl\_cert                        | --ignore-host-cn           | ssl\_cert\_ignore\_host\_cn              | #9512 |
+| ssl\_cert                        | --ignore-ocsp-errors       | ssl\_cert\_ignore\_ocsp\_errors          | #9512 |
+| ssl\_cert                        | --ignore-ocsp-timeout      | ssl\_cert\_ignore\_ocsp\_timeout         | #9512 |
+| ssl\_cert                        | --ignore-tls-renegotiation | ssl\_cert\_ignore\_tls\_renegotiation    | #9042 |
+| ssl\_cert                        | --proxy                    | ssl\_cert\_proxy                         | #8927 |
+| tcp                              | --sni                      | tcp\_sni                                 | #9347 |
+| vmware-esx-dc-runtime-tools      | --no\_vm\_tools\_ok        | vmware\_novmtools                        | #9611 |
+| vmware-esx-soap-vm-runtime-tools | --no\_vm\_tools\_ok        | vmware\_novmtools                        | #9611 |
+
+### Miscellaneous
+
+* Require GCC 7+ for building to enable C++17. #9133 #9485 #9489
+* Require CMake v2.8.12+ for building.
+  (Compatibility with older ones will be removed from a future version.) #9706
```

Comment:
"in a future **CMake** version", otherwise one could read it as Icinga 2 version (actually I have and now, when reading it for another time, I wondered that this makes no sense as there's no more compatibility with older CMake versions in Icinga 2 if we require it :sweat_smile:)